src/test/java/seedu/address/testutil/TypicalAssignments.java

```diff
@@ -0,0 +1,35 @@
+package seedu.address.testutil;
+
+import static seedu.address.testutil.TypicalShifts.SHIFT_A;
+import static seedu.address.testutil.TypicalShifts.SHIFT_B;
+import static seedu.address.testutil.TypicalShifts.SHIFT_C;
+import static seedu.address.testutil.TypicalWorkers.ALICE;
+import static seedu.address.testutil.TypicalWorkers.BOB;
+import static seedu.address.testutil.TypicalWorkers.CARL;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import seedu.address.model.assignment.Assignment;
+
+/**
+ * A utility class containing a list of {@code Assignment} objects to be used in tests.
+ */
+public class TypicalAssignments {
+
+    public static final Assignment ASSIGNMENT_A = new AssignmentBuilder().withWorker(ALICE)
+            .withShift(SHIFT_A)
```

Comment:
I think `ALICE` is unavailable during the day/time of `SHIFT_A`, perhaps not a good idea to put that as a typical assignment?

Reply:
Good catch, edited.